modules/administration/pages/auditing.adoc

```diff
@@ -46,17 +42,74 @@ We recommend that before you begin a CVE audit you refresh the data to ensure yo
 
 For more information about the patch status icons used on this page, see xref:reference:audit/audit-cve-audit.adoc[].
 
-
-For each system, the [guimenu]``Next Action`` column provides information about what you need to do to address vulnerabilities.
+For each system, the [guimenu]``Actions`` column provides information about what you need to do to address vulnerabilities.
 If applicable, a list of candidate channels or patches is also given.
 You can also assign systems to a [guimenu]``System Set`` for further batch processing.
 
-
 You can use the {productname} API to verify the patch status of your clients.
 Use the ``audit.listSystemsByPatchStatus`` API method.
 For more information about this method, see the {productname} API Guide.
 
+== OVAL
+The CVE Audit operation relies on two primary data sources: Channels and OVAL.
+These two sources provide us with metadata for conducting CVE audits, each serving a distinct purpose.
+
+1. **Channels:** Channels include the updated software packages, which include patches, and provide insights into the essential patches required to address vulnerabilities.
+
+2. **OVAL:** In contrast, OVAL data supplies information about the vulnerabilities themselves and the packages that render
+a system vulnerable to a CVE.
+
+While it is possible to conduct CVE audits using
+only channels data, synchronizing OVAL data enhances
+the accuracy of the results, particularly in cases involving zero-day vulnerabilities or partially patched
+vulnerabilities.
```

Comment:
To better support the segmentation process for creating templates for localization (l10n), we prefer a 1-line per sentence style:

```suggestion
While it is possible to conduct CVE audits using only channels data, synchronizing OVAL data enhances the accuracy of the results, particularly in cases involving zero-day vulnerabilities or partially patched vulnerabilities.
```